tests/test-cases/Group1-Docker-Commands/1-38-Docker-Exec.md

```diff
@@ -23,5 +23,41 @@ This test requires that a vSphere server is running and available
 * Step 2-6 should echo the ID given
 * Step 7 should return an error
 
+## Exec
+
 # Possible Problems:
 None
+
+# Exec Power Off test for long running Process
```

Comment:
I'm a fan of this test doc approach as it makes it easier to follow along and understand what the test is accomplishing, as opposed to simply listing commands in the markdown file. cc @mhagen-vmware

Overhauling all our test docs would be way too tedious at this point, I propose we adopt this approach for all new tests and fix up any old ones we can along the way.

Reply:
+1 to this @mhagen-vmware thoughts?